<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="setAccount_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="setPassword_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmPassword_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setAccount_LineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="setPassword_LineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPassword_LineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterBtn1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountRequire_Label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>6-11位数字、大小写字母</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordRequire_Label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数字、大小写字母</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountUnmatch_Label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>账号不符合要求</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordUnmatch_Label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码不符合要求</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
